### ISSUE TITLE
Protect against running multipe Stream/Faye clients

### ISSUE THREAD

**Issue by tschellenbach** [GetStream/stream-js#129]
Running multiple Faye clients leads to this bug:

https://github.com/GetStream/stream-js/issues/72

Which is very obscure and hard to debug. Wonder if we could provide a better error.

**Comment by thesyncim** [comment]
i believe this is related to #198:
[faye client is expecting subscription ](https://github.com/faye/faye/blob/master/src/protocol/client.js#L223) id,

also the dispatcher will bind to the subscription id as we can see [here](https://github.com/faye/faye/blob/master/src/protocol/channel.js#L104).

**Comment by tbarbugli** [comment]
@tschellenbach correct me if I wrong but AFAIK we observed this issue back when we were still using the JS implementation of Faye server-side and concluded that the problem was client-side

**Comment by tschellenbach** [comment]
Yeah this is broken with the standard Faye server. ID generation is global, so if you start 2 clients you get clashes.
